people-and-planet-ai/image-classification/requirements.txt

```diff
@@ -1,3 +1,4 @@
 Pillow==8.2.0
-apache-beam[gcp]==2.29.0
+apache-beam[gcp]==2.30.0
 google-cloud-aiplatform==1.1.1
+google-cloud-bigquery==2.18.0 # Indirect dependency, but there is a version conflict that causes pip to hang unless we constraint this.
```

Comment:
should this possibly go into a constraints.txt instead?

Reply:
There isn't a way to pass a constraints.txt file to Beam, so the Dataflow launch process would hang when staging the dependencies.

Reply:
Makes sense